gateways/cybersource/cybersource.go

```diff
@@ -100,16 +100,22 @@ func (client *CybersourceClient) Capture(request *sleet.CaptureRequest) (*sleet.
 	if err != nil {
 		return nil, err
 	}
-	capturePath := authPath + "/" + request.TransactionReference + "/captures"
+	capturePath := authPath + request.TransactionReference + "/captures"
```

Comment:
Should we fail for capture/void/refund if `TransactionReference` is empty rather than sending a request that we know is going to fail?

Reply:
This should also fix the double / mentioned here? https://boltpay.atlassian.net/browse/EN-4932

Reply:
@dbook13 Yeah that makes sense, what do you think about putting that validation logic in Sleet vs. keeping it outside? My only thought about putting it here is it would need to be repeated for all verbs for each gateway.

@ellenwei Yes, `authPath` ends in a / so this will de-dupe it.